Core/Services/InviteService.cs

```diff
@@ -55,7 +61,18 @@ public async Task ManageAsync(ManageInviteDTO manageInviteDTO, string userId)
 
             ExceptionMethods.InviteNullCheck(invite);
 
-            if (!manageInviteDTO.IsAccepted)
+            if (manageInviteDTO.IsAccepted)
+            {
+                var offers = await _offerRepository.ListAsync(new OfferSpecification.GetByTripId(invite.TripId));
```

Comment:
The length of the line is more than 100 characters, move the specification to the new line